mu4/notation/qml/MuseScore/NotationScene/PartsDialog.qml

```diff
@@ -0,0 +1,207 @@
+
+            spacing: 30
+
+            Item {
+                anchors.left: parent.left
+                anchors.right: parent.right
+
+                height: childrenRect.height
+
+                StyledTextLabel {
+                    anchors.left: parent.left
+
+                    text: qsTrc("notation", "Parts")
+
+                    font.pixelSize: 18
+                }
+
+                FlatButton {
+                    text: qsTrc("notation", "Create new part")
+
+                    anchors.right: deleteButton.left
+                    anchors.rightMargin: 8
+
+                    onClicked: {
+                        partsModel.apply()
+                        partsModel.createNewPart()
+                        root.hide()
+                    }
+                }
+
+                FlatButton {
+                    id: deleteButton
+
+                    anchors.right: parent.right
+
+                    icon: IconCode.DELETE_TANK
+
+                    onClicked: {
+                        partsModel.removeParts(selectionModel.selectedIndexes)
+                        selectionModel.clear()
+                    }
+                }
+            }
+
+            ListView {
+                id: view
+
+                height: 200
+                width: parent.width
+
+                spacing: 0
+
+                model: partsModel
+
+                boundsBehavior: Flickable.StopAtBounds
+                clip: true
+
+                readonly property int margin: 26
+
+                header: Item {
+                    width: parent.width
+                    height: childrenRect.height
+
+                    Column {
+                        anchors.bottom: parent.bottom
+                        anchors.bottomMargin: view.spacing
+
+                        width: parent.width
+
+                        spacing: 16
+
+                        StyledTextLabel {
+                            anchors.left: parent.left
+                            anchors.leftMargin: view.margin
+
+                            text: qsTrc("notation", "Name")
+                            font.capitalization: Font.AllUppercase
+                            font.pixelSize: 10
+                        }
+
+                        SeparatorLine {}
+                    }
+                }
+
+                delegate: Rectangle {
+                    id: partRect
+
+                    width: parent.width
+                    height: 40
+
+                    color: "transparent"
+                    opacity: 1
+
+                    StyledTextLabel {
+                        anchors.verticalCenter: parent.verticalCenter
+                        anchors.left: parent.left
+                        anchors.leftMargin: view.margin
+
+                        text: model.title
+
+                        font.bold: true
+                    }
+
+                    SeparatorLine { anchors.bottom: parent.bottom }
+
+                    property bool selected: false
```

Comment:
Can you just set 
property bool selected: selectionModel.hasSelection ?

Reply:
No, I can't. This is the property of a delegate. hasSelection indicated that any item is selected